### PR TITLE
added composer info

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: create_framework/http-foundation.rst [hunk 1]
```diff
@@ -121,6 +121,13 @@ To use this component, add it as a dependency of the project:
 
 Running this command will also automatically download the Symfony
 HttpFoundation component and install it under the ``vendor/`` directory.
+A ``composer.json`` and a ``composer.lock`` file will be generated as well.::
```
Comment (javiereguiluz) [inline create_framework/http-foundation.rst]: The trailing dot after `well` should be removed.

Comment (javiereguiluz) [inline create_framework/http-foundation.rst]: The double `::` "hack" can only be used when the code to highlight is PHP. In this case we need to highlight JSON, so you'll have to use a single `::` and add a `.. code-block::` directive.

**Comment by OskarStark** [comment]
@javiereguiluz done, can you please check?

File: create_framework/http-foundation.rst [hunk 2]
```diff
@@ -121,6 +121,15 @@ To use this component, add it as a dependency of the project:
 
 Running this command will also automatically download the Symfony
 HttpFoundation component and install it under the ``vendor/`` directory.
+A ``composer.json`` and a ``composer.lock`` file will be generated as well:
+
+.. code-block:: json
+
+    // composer.json
```
Comment (xabbuh) [inline create_framework/http-foundation.rst]: JSON does not have comments. So I think we should be avoid such constructs here.

Comment (OskarStark) [inline create_framework/http-foundation.rst]: shall ii use ```.. code-block:: text``` instead?

Comment (xabbuh) [inline create_framework/http-foundation.rst]: Well, I don't consider this as an issue with the syntax highlighting. It's just that someone will probably copy the code example and will then run into syntax errors because the example itself is no valid JSON.

I would rather change the example and maybe write an explanation below the code.

Comment (OskarStark) [inline create_framework/http-foundation.rst]: done @xabbuh

File: create_framework/http-foundation.rst [hunk 3]
```diff
@@ -121,6 +121,17 @@ To use this component, add it as a dependency of the project:
...
+A ``composer.json`` and a ``composer.lock`` file will be generated as well:
+
+.. code-block:: json
+
+    {
+        "require": {
+            "symfony/http-foundation": "^2.7"
+    }
+
+The code-block shows the content of the ```composer.json```, but the version may be different.
```
Comment (xabbuh) [inline create_framework/http-foundation.rst]: This should only be two backticks (two before and two after).

And I would reword it a bit:

```rst
The code block shows the content of the ``composer.json`` file (the actual
version may vary).
```

Comment (OskarStark) [inline create_framework/http-foundation.rst]: done.

File: create_framework/http-foundation.rst [hunk 4]
```diff
@@ -121,6 +121,18 @@ To use this component, add it as a dependency of the project:
...
+A ``composer.json`` and a ``composer.lock`` file will be generated as well:
+
+.. code-block:: json
+
+    {
+        "require": {
+            "symfony/http-foundation": "^2.7"
+    }
+
+``The code block shows the content of the ``composer.json`` file (the actual
```
Comment (OskarStark) [inline create_framework/http-foundation.rst]: @xabbuh you mean the backticks here?

Comment (xabbuh) [inline create_framework/http-foundation.rst]: @OskarStark Yeah, but they should only enclose the file name. So you should remove them from the beginning and the end of the paragraph now. :)

**Comment by OskarStark** [comment]
done @xabbuh

**Comment by xabbuh** [comment]
cool :+1: looks ready to be merged now

**Comment by WouterJ** [comment]
:+1:! Merged it with a few minor changes (mostly fixing the syntax error in the code example) in ae14002

**Comment by OskarStark** [comment]
Thank you @WouterJ
